frontend/public/components/modals/language-preferences-modal.tsx

```diff
@@ -38,13 +38,10 @@ const LanguagePreferencesModal = (props: LanguagePreferencesModalProps) => {
     <form onSubmit={submit} name="form" className="modal-content modal-content--no-inner-scroll">
       <ModalTitle>{t('language-preferences-modal~Edit language preferences')}</ModalTitle>
       <ModalBody>
-        <p>
-          {t(
-            'language-preferences-modal~All interface content will appear in your selected language.',
-          )}
-        </p>
         <div className="form-group">
-          <label htmlFor="language_dropdown">{t('language-preferences-modal~Language')}</label>
+          <label htmlFor="language_dropdown">
+            {t('language-preferences-modal~Select language:')}
```

Comment:
I'd probably leave this just as "Language"

We might change the user menu item and dialog title to the simpler "Select language" though. "Edit language preferences" makes it sound like there are several preferences. You're really just picking a language.

cc @beanh66 for input

Reply:
I spoke to Brie and Meghan and Meghan is going to mock some things up and share them @spadgett. (Meghan previously worked on the design for this component.)